projects/smoldot-browser-demo/package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "clean": "rm -rf dist/ && mkdir -p dist/assets",
     "copy-assets": "cp -R src/assets dist/",
-    "bundle": "yarn clean && yarn copy-assets && parcel build index.html --detailed-report",
+    "build": "yarn clean && yarn copy-assets && parcel build index.html --detailed-report",
```

Comment:
I changes this so we can run `yarn workspaces build` from the root to built all bundles